src/schemas/21P-527EZ/schema.js

```diff
@@ -279,16 +268,17 @@ let schema = {
   ['phone', 'nightPhone'],
   ['phone', 'mobilePhone'],
   ['maritalStatus'],
+  ['gender'],
   // TODO: make sure they allow dates like 2017-01-XX
-  ['date', 'dateOfMarriage'],
-  ['fullName', 'spouseFullName'],
```

Comment:
Depending on how the UI for spouses/marriages looks, we might want to add this back.